information-schema/information-schema-tidb-hot-regions-history.md

```diff
@@ -57,6 +57,8 @@ DESC tidb_hot_regions_history;
 * KEY_RATE：该 Region 内读写的 Key 数量。
 * QUERY_RATE：该 Region 内读写的 Query 数量。
 
+对于读热点，上述热点数据字段的统计周期为一个 Region 的心跳周期，由 TiKV 的配置 [`pd-heartbeat-tick-interval`](/tikv-configuration-file#pd-heartbeat-tick-interval) 决定；对于写热点，则为一个 Store 的心跳周期，由 TiKV 的配置 [`pd-store-heartbeat-tick-interval`](/tikv-configuration-file#pd-store-heartbeat-tick-interval) 决定。
```

Comment:
For qps of hot read and write, the cycle is 10s. For byte and keys for hot read, the cycle is also 10s. But for bytes and keys for hot write, it is 60s.

Reply:
Addressed. PTAL.